hw/drivers/uart/uart_bitbang/pkg.yml

```diff
@@ -28,4 +28,4 @@ pkg.deps:
     - "@apache-mynewt-core/hw/drivers/uart"
 
 pkg.init:
-    uart_bitbang_pkg_init: 'MYNEWT_VAL(UARTBB_SYSINIT_STAGE)'
+#    uart_bitbang_pkg_init: 'MYNEWT_VAL(UARTBB_SYSINIT_STAGE)'
```

Comment:
if something is removed it should be gone and not commented out.

Reply:
Can the pkg.init be made conditional on UARTBB_SYSINIT_STAGE being defined to a value? something like:
pkg.init.(UARTBB_SYSINIT_STAGE!=-1)
   uart_bitbang_pkg_init: 'MYNEWT_VAL(UARTBB_SYSINIT_STAGE)'

then if you don't want the init its just a case of defining its stage to be -1.

Reply:
Resolved by changing how I use bitbang, no more need to avoid its initialisation. Reverted to original.